hvplot/converter.py

```diff
@@ -252,26 +252,33 @@ class HoloViewsConverter(object):
                          "bottom_right", "right", "left", "top",
                          "bottom")
 
+    _default_plot_opts = {
+        'logx': False, 'logy': False, 'show_legend': True, 'legend_position': 'right',
+        'show_grid': False, 'responsive': False, 'shared_axes': True}
+
+    _default_cmaps = {
+        'linear': 'kbc_r',
+        'categorical': 'Category10',
+        'cyclic': 'colorwheel',
+        'diverging': 'coolwarm'
```

Comment:
Which 'coolwarm' is this?  It looks like the linear and cyclic defaults are taken from colorcet, so I assume it's colorcet, but it's ambiguous as matplotlib also has a coolwarm.  Colorcet's one was meant to be prefixed with cet_ to avoid confusion, but I think the prefix isn't used in some cases (?) in holoviews, so I don't know what it means here.

Reply:
I had assumed colorcet. Since matplotlib isn't supported as a backend yet we probably don't have to think about it too hard. Also I am just setting up this dict as a stop-gap until holoviews sets up a config option.

Reply:
I don't think the lack of an mpl backend will affect whether it's mpl's coolwarm; I think the way HoloViews handles colormaps depends not on the selected backend but just on what's installed and importable on the system.  I don't really know the details of what hv does here, though.